spec/motion-spec/matchers/start_with_spec.rb

```diff
@@ -0,0 +1,14 @@
+# -*- encoding : utf-8 -*-
+describe "Matcher::StartWith" do
+  it 'start_with passes when the subject starts with the given string' do
+    expect("super").to start_with("sup")
+  end
+
+  it "start_with fails when the subject doesn't start with the given string" do
+    expect_failure("\"super\" expected to start with \"key\""){ expect("super").to start_with("key") }
+  end
+
+  it 'start_with fails when the subject starts with the given string but asked not to' do
```

Comment:
Line is too long. [89/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.